cmd/antrea-controller/controller.go

```diff
@@ -50,7 +53,7 @@ const informerDefaultResync = 12 * time.Hour
 func run(o *Options) error {
 	klog.Infof("Starting Antrea Controller (version %s)", version.GetFullVersion())
 	// Create K8s Clientset, CRD Clientset and SharedInformerFactory for the given config.
```

Comment:
comment does not mention `aggregatorClient`

I think this is a good place to mention what the `aggregatorClient` is for as well. It seems that this is needed so that the aggregation layer for the Antrea API services keep working when the certificates are updated?